01.09 15:14:22 TRACE: vdl/cache.battle.join : Invalid field access : getInt
01.09 15:14:22 TRACE: vdl/cache.battle.join : 
Called from modules/VDLCache.hx line 192
Called from modules/VDLCache.hx line 148
Called from modules/VDLCache.hx line 126
Called from modules/VDLCache.hx line 75
Called from modules/VDLCache.hx line 57
Called from snipe/cache/CacheServer.hx line 487
